src/main/kotlin/org/pkl/intellij/action/PklDownloadPackageAction.kt

```diff
@@ -23,16 +23,18 @@ import com.intellij.openapi.ui.Messages
 import com.intellij.psi.PsiFile
 import org.pkl.intellij.packages.dto.PackageUri
 import org.pkl.intellij.packages.pklPackageService
+import org.pkl.intellij.toolchain.pklCli
 
 // Not registered in plugin.xml because we only use this as an intention.
 class PklDownloadPackageAction(private val packageUri: PackageUri) : IntentionAction {
   override fun startInWriteAction(): Boolean = false
 
   override fun getText(): String = "Download $packageUri"
 
-  override fun getFamilyName(): String = "Download Pkl package"
+  override fun getFamilyName(): String = "Download Pkl packages"
```

Comment:
Unrelated: make this action and PklDownloadDependencySourcesAction share the same family name.